src/JustSaying/Messaging/Monitoring/IMessageMonitor.cs

```diff
@@ -3,16 +3,69 @@
 
 namespace JustSaying.Messaging.Monitoring
 {
+    /// <summary>
+    /// Provides a mechanism for monitoring when various message actions are
+    /// performed in JustSaying. This is typically used for metrics gathering.
+    /// </summary>
     public interface IMessageMonitor
     {
+        /// <summary>
+        /// Monitors when a specific message type threw an exception while being handled
+        /// by a subscriber.
+        /// </summary>
+        /// <param name="messageType">The type of the message that was being handled.</param>
         void HandleException(Type messageType);
```

Comment:
I think in most other xml comments we stick to present tense, so maybe this could be re-phrased as:

`Called when a specific message type throws an exception while being handled by a <see cref="IHandlerAsync<T>" />.` ?